### PR TITLE
✨ Add Controlled Access field to Genomic file entity

### PR DESCRIPTION
**Pull request by parimalak**
Added Controlled Access field to Genomic file entity.
Renamed file_type to data_type.
Updated dummy data generation script,migrations and tests

### CODE REVIEW

File: migrations/versions/f80726457a72_.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Create Date: 2018-02-07 16:12:38.279955
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'f80726457a72'
+down_revision = '0cc35a21afc2'
```
Comment (dankolbman) [inline migrations/versions/f80726457a72_.py]: I believe this revision may need to be modified because of the new version added in the last merge

Comment (dankolbman) [reply migrations/versions/f80726457a72_.py]: If you change this to `b7852f8aab0a` I think you can get rid of the merge version.

File: migrations/versions/1c790b83f611_.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+"""rename file_type to data_type for genomic_file
```
Comment (znatty22) [inline migrations/versions/1c790b83f611_.py]: I know this is nit picking but can you change to: 
```python
"""
Rename file_type to data_type for genomic_file

Revision ID: 1c790b83f611
Revises: f80726457a72
Create Date: 2018-02-12 15:33:27.365798
"""
```

File: migrations/versions/f80726457a72_.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+"""Add Controlled_access field to genomic_file
```
Comment (znatty22) [inline migrations/versions/f80726457a72_.py]: Similar request as above, can you change to: 

```python 
"""
Add Controlled_access field to genomic_file

Revision ID: f80726457a72
Revises: b7852f8aab0a
Create Date: 2018-02-07 16:12:38.279955
"""
```

File: migrations/versions/f80726457a72_.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+Revision ID: f80726457a72
```
Comment (znatty22) [inline migrations/versions/f80726457a72_.py]: I think you should be able to combine these two migration script into one. On your branch, you could do: 

```
git reset origin/master migrations
rm migrations/versions/1c790b83f611_.py migrations/versions/f80726457a72_.py
# Make sure your dataservice containers are running, and if not run:
docker-compose up -d
# Generate new migration script
flask db migrate
``` 
That should generate a new migration script. Make sure you update the docstring to include both the field changes

**Review by dankolbman — APPROVED** [review]

**Review by znatty22 — APPROVED** [review]
👍

**Comment by parimalak** [comment]
✨ Update fields(controlled_access, data_type)  in Genomic file entity

**Pull request merged**